frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/RebaseOnParentBackgroundable.java

```diff
@@ -46,13 +46,13 @@ public RebaseOnParentBackgroundable(String title, GitRepository gitRepository,
       IGitMacheteRepositorySnapshot gitMacheteRepositorySnapshot,
       INonRootManagedBranchSnapshot branchToRebase,
       boolean shouldExplicitlyCheckout) {
-    super(gitRepository.getProject(), title);
+    super(gitRepository.getProject(), title, /* name */ "rebase");
```

Comment:
I see the inconsistent naming of a variable here: `/* name */`. IMHO it can be dropped.